youtube_dl/extractor/vidlii.py

```diff
@@ -83,9 +126,9 @@ def _real_extract(self, url):
             r'duration\s*:\s*(\d+)', webpage, 'duration', fatal=False))
 
         view_count = int_or_none(self._search_regex(
-            (r'<strong>(\d+)</strong> views',
-             r'Views\s*:\s*<strong>(\d+)</strong>'),
-            webpage, 'view count', fatal=False))
+            (r'<strong>([\d,]+)</strong> views',
+             r'Views\s*:\s*<strong>([\d,]+)</strong>'),
+            webpage, 'view count', fatal=False).replace(",", ""))
```

Comment:
If these `,`s are thousands separators, might they be `.`s for some locales (plainly not a decimal point for a count)?
```suggestion
        view_count = str_to_int(self._html_search_regex(
            (r'<strong>([\d,.]+)</strong> views',
             r'Views\s*:\s*<strong>([\d,.]+)</strong>'),
            webpage, 'view count', fatal=False))
```
(and `from ..utils import str_to_int` at the top).